CHANGELOG.md

```diff
@@ -7,7 +7,14 @@ reference the gitlab/github issue that is related to the change.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v10.4.0] - TBD
+## [v10.5.0] - TBD
+
+- `#155` cim: Check if impacted service has subscription id before calling api
+- `#156` cim: Add html parser to form summary field for email examples
```

Comment:
Maybe put these in a "Surf specific:" section below the other generic changes, similar to v10.4.0

Reply:
Will do, had completely glanced over it